dist/cornerstoneTools.js

```diff
@@ -1,4 +1,8 @@
-/*! cornerstoneTools - v0.6.2 - 2015-07-31 | (c) 2014 Chris Hafey | https://github.com/chafey/cornerstoneTools */
+<<<<<<< HEAD
```

Comment:
Everything looks good except for this! Looks like a rebase / merge artifact. Try running grunt again and committing the result, then it looks good to merge!